src/AbstractAlgebra.jl

```diff
@@ -15,7 +15,7 @@ using Preferences
 using Test # for "interface-conformance" functions
 
 import GroupsCore
-import GroupsCore: gens, ngens, order, mul!, istrivial
+import GroupsCore: gens, order, mul!, istrivial
```

Comment:
I guess we need to decide how to order this WRT to the GroupsCore removal PR...

Reply:
There already is a comment somewhere below on how to chamge this PR after the group score one has been merged. So this will get a small update after the merge of #1528